app.py

```diff
@@ -29,7 +42,11 @@ def add_animal() -> tuple[Response, int]:
     new_animal = Animal(
         animal_type=data.animal_type,
         name=data.name,
-        birth_date=data.birth_date
+        birth_date=data.birth_date,
+        age = data.age,
```

Comment:
you shouldn't save age as field in database, because age value can be changed, and you will not change it in DB. That's why age should be a calculated field, which will be calculated every time you using animal pydantic model.